src/cmds/core/identify.py

```diff
@@ -22,6 +22,37 @@ class IdentifyCog(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    class VerifyError(discord.ui.view):
```

Comment:
`discord.ui.view` is not a class, so you cannot extend. Maybe you wanted to write `discord.ui.view.View`. In that case better import it `from discord.ui.view import View` and use it.